sharepoint/3.2/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,396 @@
+= Microsoft SharePoint Connector 3.2
+:page-aliases: connectors::sharepoint/sharepoint-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Microsoft SharePoint (Microsoft SharePoint Connector) supports both SharePoint 2013 and SharePoint Online for use in the cloud and on-premises. This connector enables you to manage content and document management, to access  intranet and extranet portals and websites, to collaborate with, and to search your entire enterprise. This connector also enables integration with SharePoint through its REST API.
+
+Release Notes: xref:release-notes::connector/sharepoint-connector-release-notes-mule-4.adoc[Microsoft SharePoint Connector Release Notes] +
```

Comment:
We can remove these links since they are in the info box now. We just have a sentence now that says:  For compatibility information and fixed issues, see the Microsoft Sharepoint Connector Release Notes.